.stylelintrc

```diff
@@ -1,4 +1,5 @@
 {
+    "customSyntax": "postcss-less",
```

Comment:
stylelint no longer provides syntaxes for anthing but vanilla css as of version 14. We have to provide a syntax.